Databases.sql

```diff
@@ -0,0 +1,2 @@
+create database Belfast_team2_MylesS;
```

Comment:
Can you make the SQL parts of this upper case, remove the use statement and add a blank line at the end